assets/js/randomSite.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * This looks for elements with the 'site' class which in this case are all of the
+ * anchor links for the sites. This could certainly be made more robust with
+ * some sanity checks, but should work in this context.
+ */
+function randomSite() {
+  const sites = document.getElementsByClassName('site')
+
+  if (sites.length > 0) {
+    const choiceIndex = Math.floor(Math.random() * (sites.length - 1))
```

Comment:
Because of the -1 the last item will never get selected. 
Since random() never returns 1, only 0.9999… and you also use floor floor it will never reach sites.length

Reply:
Yep. You are correct, this is a bug. Thanks for checking it over!

Filed a new PR at https://github.com/kevquirk/512kb.club/pull/545